src/Locale/en/Resource.json

```
@@ -0,0 +1,11 @@
+{
+  "create_title": "Create Resource Request",
```

@cp-Coder the translation key `create_title` is too generic to be used for a specific `Create Resource Request` and could cause confusion and/or conflicts with other keys.

```suggestion
  "create_resource_request": "Create Resource Request",
```